File: IM.web1/ClientApp/src/app/Models/member.ts
import { BaseEntity } from "./baseEntity";
import { MemberSkill } from "./memberSkill";

export class Member  extends BaseEntity{
  name: string = '';
  countryId: number=0;
  cityId: number=0 ;
  resume: string = '';
  dateOfBirth: any;
  memberSkills:MemberSkill[]=[];

  countryName: string = '';
  cityName?: string = '';
  languageSkills:string='';
}

File: IM.Core/Entities/Member.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace IM.Core.Entities
{
    public class Member : BaseEntity
    {
        public Member()
        {
            MemberSkills = new HashSet<MemberSkill>();
        }

        
        public string Name { get; set; } = null!;
        public long CountryId { get; set; }
        public string Resume { get; set; } = null!;
        public DateTime DateOfBirth { get; set; }
        public long CityId { get; set; }
        

        public virtual City City { get; set; } = null!;
        public virtual Country Country { get; set; } = null!;
        public virtual ICollection<MemberSkill> MemberSkills { get; set; }
    }
}
